src/Analyzers/CSharp/Tests/ForEachCast/ForEachCastTests.cs

```diff
@@ -1096,8 +1096,8 @@ public static void M(Regex regex, string text)
     }
 }
 ";
-            await TestAlwaysAsync(test, test);
-            await TestWhenStronglyTypedAsync(test, test);
+            await TestAlwaysAsync(test, test, ReferenceAssemblies.Net.Net80);
+            await TestWhenStronglyTypedAsync(test, test, ReferenceAssemblies.Net.Net80);
```

Comment:
don't change existing tests, write new ones.  for the existing tests show the new behaviors with the same args :)

Reply:
@CyrusNajmabadi These are the new tests.

Reply:
weird! it showed up strange in the diff. thanks!